.github/workflows/unittest.yml

```diff
@@ -0,0 +1,30 @@
+name: unit-test
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches: [next, qa, main]
+
+jobs:
+  linux:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: conda-incubator/setup-miniconda@v2
+      with:
+        auto-update-conda: true
+        mamba-version: "*"
+        environment-file: conda_environment.yml
+    - name: Install testing requirements
+      run: mamba env update --name test --file conda_development.yml
+    - name: Install iMars3dv2
+      run: pip install -e .
```

Comment:
Things actually work a bit better if this is `conda develop .` which is functionally the same thing

Reply:
switched to suggested method